spot_driver_cpp/include/spot_driver_cpp/conversions/kinematic_conversions.hpp

```diff
@@ -0,0 +1,81 @@
+    const bosdyn_msgs::msg::InverseKinematicsRequestFixedStance& ros_msg,
+    bosdyn::api::spot::InverseKinematicsRequest::FixedStance& proto);
+
+void convert_bosdyn_msgs_inverse_kinematics_request_on_ground_plane_stance_to_proto(
+    const bosdyn_msgs::msg::InverseKinematicsRequestOnGroundPlaneStance& ros_msg,
+    bosdyn::api::spot::InverseKinematicsRequest::OnGroundPlaneStance& proto);
+
+void convert_bosdyn_msgs_inverse_kinematics_request_one_of_stance_specification_to_proto(
+    const bosdyn_msgs::msg::InverseKinematicsRequestOneOfStanceSpecification& ros_msg,
+    bosdyn::api::spot::InverseKinematicsRequest& proto);
+
+void convert_bosdyn_msgs_inverse_kinematics_request_wrist_mounted_tool_to_proto(
+    const bosdyn_msgs::msg::InverseKinematicsRequestWristMountedTool& ros_msg,
+    bosdyn::api::spot::InverseKinematicsRequest::WristMountedTool& proto);
+
+void convert_bosdyn_msgs_inverse_kinematics_request_body_mounted_tool_to_proto(
+    const bosdyn_msgs::msg::InverseKinematicsRequestBodyMountedTool& ros_msg,
+    bosdyn::api::spot::InverseKinematicsRequest::BodyMountedTool& proto);
+
+void convert_bosdyn_msgs_inverse_kinematics_request_one_of_tool_specification_to_proto(
+    const bosdyn_msgs::msg::InverseKinematicsRequestOneOfToolSpecification& ros_msg,
+    bosdyn::api::spot::InverseKinematicsRequest& proto);
+
+void convert_bosdyn_msgs_inverse_kinematics_request_tool_pose_task_to_proto(
+    const bosdyn_msgs::msg::InverseKinematicsRequestToolPoseTask& ros_msg,
+    bosdyn::api::spot::InverseKinematicsRequest::ToolPoseTask& proto);
+
+void convert_bosdyn_msgs_inverse_kinematics_request_tool_gaze_task_to_proto(
+    const bosdyn_msgs::msg::InverseKinematicsRequestToolGazeTask& ros_msg,
+    bosdyn::api::spot::InverseKinematicsRequest::ToolGazeTask& proto);
+
+void convert_bosdyn_msgs_inverse_kinematics_request_one_of_task_specification_to_proto(
+    const bosdyn_msgs::msg::InverseKinematicsRequestOneOfTaskSpecification& ros_msg,
+    bosdyn::api::spot::InverseKinematicsRequest& proto);
+
+void convert_bosdyn_msgs_inverse_kinematics_request_to_proto(const bosdyn_msgs::msg::InverseKinematicsRequest& ros_msg,
+                                                             bosdyn::api::spot::InverseKinematicsRequest& proto);
+
+///////////////////////////////////////////////////////////////////////////////
+// Protobuf to ROS.
+
+void convert_proto_to_bosdyn_msgs_request_header(const bosdyn::api::RequestHeader& proto,
+                                                 bosdyn_msgs::msg::RequestHeader& ros_msg);
+
+void convert_proto_to_bosdyn_msgs_common_error(const bosdyn::api::CommonError& proto,
+                                               bosdyn_msgs::msg::CommonError& ros_msg);
+
+void convert_proto_to_bosdyn_msgs_response_header(const bosdyn::api::ResponseHeader& proto,
+                                                  bosdyn_msgs::msg::ResponseHeader& ros_msg);
```

Comment:
These conversions seem fairly generic. Could they go in a different header than `kinematic_conversions.hpp`?

Reply:
This makes sense, I moved them into the `common_conversion` file.